data/src/main/scala/ch.epfl.scala.index.data/central/CentralMissing.scala

```diff
@@ -93,56 +82,45 @@ class CentralMissing(paths: DataPaths)(implicit val system: ActorSystem) {
   import system.dispatcher
 
   private val mavenSearchConnectionPool: Flow[
-    (HttpRequest, ArtifactRequest),
-    (Try[HttpResponse], ArtifactRequest),
+    (HttpRequest, SearchRequest),
+    (Try[HttpResponse], SearchRequest),
     Http.HostConnectionPool
   ] = {
 
     Http()
-      .cachedHostConnectionPoolHttps[ArtifactRequest]("search.maven.org")
+      .cachedHostConnectionPoolHttps[SearchRequest]("search.maven.org")
       .throttle(
         elements = 100,
-        per = 1.minute,
-        maximumBurst = 50,
-        mode = ThrottleMode.Shaping
+        per = 1.minute
       )
   }
 
-  private def request(gaRequest: ArtifactRequest): HttpRequest = {
-    import gaRequest._
-
+  private def toHttp(request: SearchRequest): HttpRequest = {
     HttpRequest(
       uri = Uri("/solrsearch/select").withQuery(
         Query(
-          "q" -> s"""g:"$groupId" AND a:"$artifactId" """,
+          "q" -> s"""g:"${request.groupId}"""",
+          "rows" -> "20000",
```

Comment:
Should we also handle pagination for organizations having more than 20k artifacts?

Reply:
Yes, it would be nice to handle pagination.